recipes/win10toast/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "win10toast" %}
+{% set version = "0.9" %}
```

Comment:
How was this version determined?

Reply:
PyPI: https://pypi.org/project/win10toast/0.9/#files

Reply:
And the [source](https://github.com/jithurjacob/Windows-10-Toast-Notifications/blob/9d52b73f1af6c60162cf09b99269c4f7b13cdb00/setup.py#L23).